README.md

```diff
@@ -50,7 +50,16 @@ and deploy the code.
 
 ### Pre-requisites
 
-1. To run the scripts, Windows users require [Powershell 7.2 (LTS)](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows) or above. Alternatively, you can use a bash terminal using [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install). macOS users can use a bash terminal.
+1. To run the scripts, Windows users require Powershell 7.2 (LTS) or above. Alternatively, you can use a bash terminal using [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install). macOS users can use a bash terminal.
+
+1. [Install PowerShell](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows)
+    Run the following to verify that you're running the latest PowerShell
+
+    ```ps1
+    $PsVersionTable
+    ```
+
+    > or install with `winget install -e --id Microsoft.PowerShell`
```

Comment:
You're introducing winget for the first time here - I wouldn't assume all your readers know what it is. I would recommend we at least link off to installation instructions for winget. Maybe "If you would like to use the Windows Package Manager winget (get it [here](https://learn.microsoft.com/en-us/windows/package-manager/winget/)) you can install it with ...

Reply:
from new win11 VM
`The term 'winget' is not recognized as the name of a cmdlet`

Reply:
Removed the winget install and will leverage the link as-is for readers to choose the install process that works for them as outlined in https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.3